lib/igb/igb.c

```diff
@@ -1837,7 +1837,10 @@ struct timespec timespec_addns(struct timespec *a, unsigned long addns)
 	return *a;
 }
 
-#define TS2NS(ts) (((ts).tv_sec*1000000000)+(ts).tv_nsec)
+static inline u_int32_t TS2NS(struct timespec ts)
+{
+    return (((ts).tv_sec*1000000000)+(ts).tv_nsec);
+}
```

Comment:
Again - was this required for the 32-bit build? Perhaps a separate commit would be better. Also, the (ts).tv_sec could be replaced with ts.tv_sec if you are now using an inline function.